gammapy/datasets/make.py

```diff
@@ -83,32 +83,55 @@ def sigma_energy_theta(energy, theta, sigma):
 
 
 def make_test_observation_table(observatory_name='HESS', n_obs=10,
-                                datestart=None, dateend=None,
+                                az_range=Angle([0, 360], 'degree'),
+                                alt_range=Angle([45, 90], 'degree'),
+                                date_range=(Time('2010-01-01T00:00:00',
+                                                 format='isot', scale='utc'),
```

Comment:
Let's adopt the Astropy default for Gammapy and trust that they won't change it.
(It's highly unlikely ... it would break a lot of user codes if they did ... satellites falling out of the sky ... doomsday.)

I wanted to write "the format is not important", because it's not used for computations.